### PR TITLE
this is it, you made it!

### PR DESCRIPTION
**Pull request by aaronj1335**
- [ ] @miketaylr is this sufficiently libelous?
- [x] @lawnsea how many grammar/spelling mistakes?
- [ ] @andycouch are you folks actually up for hosting this month?

let's also loop in @zachleat to confirm any dark web allegations.

### CODE REVIEW

**Comment by zachleat** [comment]
me nodding emphatically

File: _posts/2016-05-10-meedup.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+when: 2016-05-17T19:30:00-05:00
+---
+
+{% assign speakr = 'Mike Taylor' %}
+{% assign handle = 'miketaylr' %}
+{% assign twiturl = 'http://twitter.com/miketaylr' %}
+{% assign avi = 'https://avatars1.githubusercontent.com/u/67283' %}
+{% assign sponsurl = 'https://www.homeaway.com/info/about-us/careers-1/career-opportunities' %}
+
+This month we've got [dark web expert][] and [Gotye fan][] [<strike>Miike</strike> Mike <strike>Snow</strike> Taylor]({{ twiturl }}) in to discuss the nuances of writing JavaScript that actually works in world-wide web browsing machines.  Come out and join us next Tuesday <x-date>{{ site.categories.meetups[0].when | date_to_xmlschema }}</x-date>!
```
Comment (miketaylr) [inline _posts/2016-05-10-meedup.md]: `[Gotye fan][]`

h8 u

File: _posts/2016-05-10-meedup.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+title: Making Browsers Compatible with Web JavaScript
+author: aaron
+layout: post
+categories:
+- posts
+- meetups
+when: 2016-05-17T19:30:00-05:00
+---
+
+{% assign speakr = 'Mike Taylor' %}
```
Comment (miketaylr) [inline _posts/2016-05-10-meedup.md]: speakr is appropriately web 2.0. i approve.

**Comment by miketaylr** [comment]
:shipit:

File: _posts/2016-05-10-meedup.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  <div>
+    <a href="{{ twiturl }}"><strong>{{ speakr }}</strong></a>
+
+    is a Web Compatibility Engineer at Mozilla. He works from home in Austin,
+    TX.  Once upon a time he worked at Opera Software as a Web Opener, whatever
+    that means.
+  </div>
+</div>
+
+[Frog][] is hosting us this month. You folks are awesome as always. We'll also be enjoying food and beverage this thanks to [Home Away][sponsor]
```
Comment (lawnsea) [inline _posts/2016-05-10-meedup.md]: - s/beverage/beverages
- s/this thanks/this time thanks

**Comment by lawnsea** [comment]
LGTM other than the above typos. 🚢 that 💩

**Pull request closed without merging**